sdp-utils.c

```diff
@@ -1498,6 +1498,14 @@ janus_sdp *janus_sdp_generate_answer(janus_sdp *offer, ...) {
 			if(!do_data || data > 1) {
 				/* Reject */
 				am->port = 0;
+				/* Add the format anyway, to keep Firefox happy */
+				GList *fmt = m->fmts;
+				if(fmt) {
+					char *fmt_str = (char *)fmt->data;
+					if(fmt_str)
+						am->fmts = g_list_append(am->fmts, g_strdup(fmt_str));
+					JANUS_LOG(LOG_FATAL, "[answer] %p, %p\n", am, am->fmts);
```

Comment:
Is the log statement a debugging leftover? I'm not sure the pointers need to be logged as fatal message...

Reply:
Argh, good catch, I thought I had gotten rid of them all... It was just lazy debugging on my end :slightly_smiling_face:

Reply:
Fixed :+1: